advisories/github-reviewed/2022/05/GHSA-6hvf-xvwm-vrw4/GHSA-6hvf-xvwm-vrw4.json

```diff
@@ -18,7 +18,7 @@
     {
       "package": {
         "ecosystem": "Maven",
```

Comment:
Target language is C++ - not quite sure what would be the proper ecosystem/name combination in the present situation.